engine/api/group.go

```diff
@@ -210,6 +215,9 @@ func addGroupHandler(w http.ResponseWriter, r *http.Request, db *gorp.DbMap, c *
 	}
 	defer tx.Rollback()
 
+	if strings.HasSuffix(g.Name, sdk.DefaultGroupOnProjectSuffix) {
+		return sdk.WrapError(sdk.ErrWrongRequest, "You can create a group with ending with %s", sdk.DefaultGroupOnProjectSuffix)
```

Comment:
i don't understand, if the group ends with the PRJ suffix, we return an error telling him he can do it ?